packages/commonwealth/client/scripts/views/pages/AdminPanel/RefreshCustomDomainTask.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
+import { notifyError } from '../../../controllers/app/notifications';
+import useRefreshCustomDomainQuery from '../../../state/api/communities/refreshCustomDomain';
```

Comment:
there is an index file in api/communities/index.ts
please add both query and mutation you created there